src/plugins/data/public/field_formats/field_formats_service.ts

```diff
@@ -44,6 +46,10 @@ export class FieldFormatsService {
   }
 
   public start() {
+    this.fieldFormatsRegistry.deserialize = deserializeFieldFormat.bind(
```

Comment:
?Why not add the `deserialize` method to `FieldFormatsRegistry` instead of binding?

Reply:
its only available on the client, the class is common at the moment. i could add a class extending it i guess but as its temporary solution (until we can make deserialization work on the server) i used binding.